packages/flutter_tools/lib/src/android/android_sdk.dart

```diff
@@ -216,10 +216,13 @@ class AndroidNdk {
           .split('\n')
           .map<String>((String line) => line.trim())
           .where((String line) => line.isNotEmpty);
-      final Map<String, String> properties = Map<String, String>.fromIterable(
-          propertiesFileLines.map<List<String>>((String line) => line.split(' = ')),
-          key: (dynamic split) => split[0],
-          value: (dynamic split) => split[1]);
+      final Map<String, String> properties = <String, String>{};
+      for (String line in propertiesFileLines) {
+        final List<String> parts = line.split(' = ');
+        if (parts.length == 2) {
```

Comment:
Do you want to at least print a warning if it's not parsing correctly?  I know we don't want to crash, but there should be some indication that something is amiss...

Reply:
Done